titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 From line 15 to 16_
```python
# Show see plot
# Show wee plot
```
Multil line comment

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/10)._